teams/ruby_services.md

```diff
@@ -129,15 +129,14 @@ test_story = build(:story, :published, in_print: false)
 
 The issue we have found is that as time goes on, the required conditions of the object may change, and then there is contention as to whether the existing trait gets updated or the test code.
 
-To avoid this our agreed pattern is that *all object setup should be in a trait*.
+Every factory should include the essential data by default. If a test needs specific data that differs from the `default` status of the object, it should use a trait to define the data.
+For example, a test for the presence of a specific `title` should define a new trait with the expected title setup and test for it rather than use the base object. The idea is that basic data allow validations of the object through the different transition states but **will not** guarantee that the data will be consistent over time unless placed in a trait.
```

Comment:
```suggestion

For example, a test for the presence of a specific `title` should define a new trait with the expected title setup and test for it rather than use the base object. The idea is that basic data allow validations of the object through the different transition states but **will not** guarantee that the data will be consistent over time unless placed in a trait.
```

Not sure if this suggestion will work, but I think we are missing an extra line break here – had a peek at the rendered document and it mashes the two paragraphs together.

Reply:
oh strange! Will apply :)